spec/controllers/campaigns_controller_spec.rb

```diff
@@ -64,16 +64,27 @@
     let(:user) { create(:user) }
     let(:admin) { create(:admin) }
     let(:campaign) { create(:campaign) }
+    let(:campaign_user) do
+      create(:campaigns_user, user_id: user.id, campaign_id: campaign.id,
```

Comment:
You'll need to create a new factory in `spec/factories`. Just follow the example from `spec/factories/campaigns_courses.rb` and do `annotate`.